src/core/src/op/result.cpp

```diff
@@ -71,6 +71,10 @@ bool Result::constant_fold(OutputVector& output_values, const OutputVector& inpu
     return false;
```

Comment:
Can be removed as Base implementation will return false.

Reply:
done